dataland-inbound-proxy/custom_errors.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en";>
+<head>
+    <meta charset="UTF-8">
+    <title>Sorry an error occurred</title>
+</head>
+<body style="margin:0px;">
+<div class="bg-black" style="height:4rem" style="padding-top:0rem; padding-left:0rem;">
+    <div style="padding-left: 1rem; padding-top: 1rem;">
```

Comment:
Doppelte Style tags sind glaube ich keine gute Idee.

Reply:
Zudem würde ich persönlich glaube ich lieber den gesamten Style in Klassen auslagern und auf inline-style verzichten

Reply:
Styling tags komplett entfernt und durch classes ersetzt.